docs/designs/sanitize-html.md

```diff
@@ -0,0 +1,153 @@
+# Sanitize HTML
+
+[Feature 178608](https://dev.azure.com/ceapex/Engineering/_workitems/edit/178608/)
+
+## Overview
+
+Content authors can embed arbitrary HTML tags to markdown documents. These HTML tags could contain `<script>` tag that allows arbitrary script execution, or `style` attribute that compromises site visual experience.
+
+### Goals
+
+- Remove HTML tags and attributes from user content that may alter site style
+- Remove HTML tags and attributes from user content that may allow javascript execution
+- Apply HTML sanitization consistently for all user content
```

Comment:
Does that include all reference contents and YAML contents?

Reply:
If including reference and YAML contents cost much effort, can we consider only including Markdown files as first phase?

Reply:
The current HTML post processor runs against all user contents, including YAML or reference, this is 0 cost 😄